proxy/http2/Http2Stream.cc

```diff
@@ -532,6 +534,16 @@ Http2Stream::update_write_request(IOBufferReader *buf_reader, int64_t write_len,
     if (bytes_avail > num_to_write) {
       bytes_avail = num_to_write;
     }
+
+    if (static_cast<Http2ClientSession *>(parent)->write_buffer_size() > HTTP2_MAX_BUFFER_USAGE ||
```

Comment:
Shouldn't we limit `bytes_avail` to `client_rwnd`?

Reply:
I worry that the client will have a large receive window and use up a lot of memory on the server.  It is better to limit it at the server.